book.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+let myLibrary = [];
+
+function Book(author, title, pageCount, hasRead) {
+  const book = {
+    author,
+    title,
+    pageCount,
+    hasRead,
+  };
+  return book;
+}
+
+function clearDisplay() {
+  const bookholder = document.getElementById('bookholder');
+  bookholder.innerHTML = '';
+}
+
+function displayLibrary() {
+  clearDisplay();
+}
+
+function addBookToLibrary() {
```

Comment:
- [ ] <b>OPTIONAL:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```
- [ ] <b>OPTIONAL:<b/> You are missing one feature of changing book status. This feature is mandatory in project specifications. I strongly recommend implementing it.